tests/framework/http.py

```diff
@@ -16,7 +17,8 @@ class Session(requests_unixsocket.Session):
 
     def __init__(self):
         """Create a Session object and set the is_good_response callback."""
-        super().__init__()
+        super(Session, self).__init__()
```

Comment:
Can we also explain why we're setting `pool_connections` to `1`? It would be useful for the brave souls that venture into this area in the future :)